pkg/updatechecker/updatechecker.go

```diff
@@ -26,13 +30,14 @@ import (
 // jobs maps app ids to their cron jobs
 var jobs = make(map[string]*cron.Cron)
 var mtx sync.Mutex
+var store = storepkg.GetStore()
```

Comment:
could there be a race condition where this package is initialized before the store package and it would get the wrong store or maybe nil?